src/vma/util/sys_vars.h

```diff
@@ -541,37 +541,37 @@ extern mce_sys_var & safe_mce_sys();
 #define MCE_DEFAULT_TCP_MAX_SYN_RATE                	(0)
 #define MCE_DEFAULT_TX_NUM_SEGS_TCP			(1000000)
 #define MCE_DEFAULT_TX_NUM_BUFS				(200000)
-#define MCE_DEFAULT_TX_NUM_WRE				(3000)
-#define MCE_DEFAULT_TX_NUM_WRE_TO_SIGNAL		(64)
+#define MCE_DEFAULT_TX_NUM_WRE				(256)
+#define MCE_DEFAULT_TX_NUM_WRE_TO_SIGNAL		(4)
 #define MCE_DEFAULT_TX_MAX_INLINE			(220) //224
 #define MCE_DEFAULT_TX_BUILD_IP_CHKSUM			(true)
 #define MCE_DEFAULT_TX_MC_LOOPBACK			(true)
 #define MCE_DEFAULT_TX_NONBLOCKED_EAGAINS		(false)
 #define MCE_DEFAULT_TX_PREFETCH_BYTES			(256)
-#define MCE_DEFAULT_TX_BUFS_BATCH_UDP			(8)
-#define MCE_DEFAULT_TX_BUFS_BATCH_TCP			(16)
+#define MCE_DEFAULT_TX_BUFS_BATCH_UDP			(1)
```

Comment:
This sort of change will hurt VMA's packet rate results for OOB, while improving latency. 
Just making sure we're all align with that.